### ISSUE TITLE
Why is the pixel loss not overfitting on a small set of data.

### ISSUE THREAD

**Issue by Leroy357** [tinghuiz/SfMLearner#86]
I tried training the network on a subset of kitti(18G), but the pixel loss is not decreasing after two iterations. As a sanity check, I train the model on just two image sequences, and the prediction on that single image is still not as good as the pretrained model. I'm wondering what could be wrong. I haven't modified the code yet.
